client/webserver/site/src/js/settings.ts

```diff
@@ -319,8 +310,6 @@ export default class SettingsPage extends BasePage {
   /* Called after successful registration to a DEX. */
   async registerDEXSuccess () {
     const page = this.page
-    page.dexAddr.value = ''
-    this.clearCertFile()
```

Comment:
@martonp Would you comment on this?  You added this in https://github.com/decred/dcrdex/pull/1135

There's also a `clearCertFile` in forms.ts.

I'm not clear on the root cause of this bug or why the fix is appropriate.  @vctt94 Can you comment on what you observed going wrong, and why we do not need to clear this?

Reply:
In `forms.tmpl`, `dexAddr` used to be the id of the element, but it got updated to be a `data-tmpl` at some point, so it wasn't getting loaded into the `page` object on the settings page.